### PR TITLE
MGMT-2770 add inventory to installing-in-progress

### PR DESCRIPTION
**Pull request by rollandf**
Temporary add get inventory to installing-in-progress state
to get change of the formatted disks in the inventory model (bootable)

### CODE REVIEW

**Comment by rollandf** [comment]
/hold
Verifying with bootable disks in test-infra

**Comment by rollandf** [comment]
@filanov

File: internal/host/instructionmanager.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func NewInstructionManager(log logrus.FieldLogger, db *gorm.DB, hwValidator hard
 			models.HostStatusDiscovering:              {[]CommandGetter{inventoryCmd, downloadInstallerCmd}, defaultNextInstructionInSec},
 			models.HostStatusPendingForInput:          {[]CommandGetter{inventoryCmd, connectivityCmd, freeAddressesCmd, dhcpAllocateCmd}, defaultNextInstructionInSec},
 			models.HostStatusInstalling:               {[]CommandGetter{installCmd, dhcpAllocateCmd}, defaultBackedOffInstructionInSec},
-			models.HostStatusInstallingInProgress:     {[]CommandGetter{dhcpAllocateCmd}, defaultNextInstructionInSec},
+			models.HostStatusInstallingInProgress:     {[]CommandGetter{inventoryCmd, dhcpAllocateCmd}, defaultNextInstructionInSec},
```
Comment (filanov) [inline internal/host/instructionmanager.go:82]: Please write a todo here that explain that this is a temporary solution until format command is moved to a different state.

Comment (rollandf) [reply internal/host/instructionmanager.go]: Done

**Review by filanov — APPROVED** [review]

**Pull request merged**